### DISCUSSION TITLE
Is it possible to trigger the image picker from outside the widget?

### DISCUSSION THREAD

**Discussion by leekaiwei** [flutter-form-builder-ecosystem/form_builder_image_picker#57] [Q&A] [answered]
So the default behaviour is to tap on the `FormBuilderImagePicker` widget to trigger opening of the image picker. Is there anyway for example to place a button outside of the widget to also trigger this behaviour? Or should I be using just the `image_picker` library and manually get that working with `form_builder`?

**Comment by deandreamatias** [comment, marked as answer]
Use a image_picker and form_builder separately in this case. Can set a form field value on your FormBuilderImagePicker
